specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/MigrateFromSmartDetections.json

```diff
@@ -0,0 +1,304 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-01-01-preview",
+    "title": "Azure Alerts Management Service Resource Provider",
+    "description": "APIs for managing a migration from Smart Detection to smart alerts."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection": {
+      "post": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Starts an asynchronous migration process of Smart Detection to smart alerts in an Application Insights resource.",
+        "operationId": "MigrateFromSmartDetection_StartMigration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "smartDetectionMigrationRequest",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SmartDetectionMigrationRequest"
+            },
+            "description": "The parameters controlling the migration process."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Migration request was accepted. Processing will start promptly.",
+            "schema": {
+              "$ref": "#/definitions/MigrationStatusResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MigrationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Migration Triggering": {
+            "$ref": "./examples/migrateFromSmartDetectionStartMigration.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true, [](start = 8, length = 36)

According to our RPC, you still need to return a terminal state - that's why you are getting the linting error, see [here](https://github.com/Azure/azure-rest-api-specs/blob/44ec11c74634f1afa5bbe0d3f2678b7f819ebf8e/specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimtenant.json#L340) to find an example. 

The way it should work is as follows:

1. First POST call returns 202
2. Location header contains a URL to monitor the operation progress
3. If another POST call is invoked to the same resource id (URL path) using the same exact body, your service should treat it as a no-op, returns a response code = 200 and the response content should be your resource object - this is the reason why you need to return a terminal state

Hope it makes sense.